tests/test_fields.py

```diff
@@ -90,16 +108,16 @@ def test_value_returned_is_not_bytea(self):
         EncryptedModelFactory.create()
 
         instance = self.model.objects.get()
-        self.assertIsInstance(instance.digest_field, str)
-        self.assertIsInstance(instance.hmac_field, str)
+        self.assertIsInstance(instance.digest_field, unicode)
+        self.assertIsInstance(instance.hmac_field, unicode)
```

Comment:
Might be better to take the "python 3 first" approach, and stick with `str`. Alternatively, how about something that can't be confused: `text_type` vs `bytes_type`, or similar. I think that's what `six` does, no?